aibolit/patterns/return_null/return_null.py

```diff
@@ -20,31 +20,41 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-import javalang
-from collections import defaultdict
-
+from typing import List
 from aibolit.utils.ast_builder import build_ast
+from aibolit.utils.ast import AST, ASTNodeType
 
 
 class ReturnNull:
 
     def __init__(self):
         pass
 
-    def value(self, filename: str):
+    def get_return_args(self, node: int, tree: 'AST') -> bool:
```

Comment:
You shouldn't use quotes here `'AST'`. Quotes are used, when class you use is not defined yet.
This function name says it gets something, but it is actually check something. Please change the name.
You might make it private as well.

Reply:
fixed